distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -179,8 +183,7 @@ type Vector a
     new_builder : Integer -> Builder
     new_builder (capacity=10) = Builder.new capacity
```

Comment:
Why have the `new_builder` method at all, when we want to hide it? If users are supposed to use `Vector.build` in 99% of cases, then let's remove `new_builder` and keep just `Builder.new`! Copy this documentation to `Builder.new` then.

Reply:
Done.